src/igvfd/audit/item.py

```diff
@@ -53,3 +57,58 @@ def audit_item_schema(value, system):
         )
         )
         yield AuditFailure(category, detail, level='INTERNAL_ACTION')
+
+
+@audit_checker('Item', frame='object')
+def audit_item_status(value, system):
+    if 'status' not in value:
+        return
+
+    # objects with these statuses are not audited
+    if value['status'] in ['revoked', 'disabled', 'deleted', 'replaced']:
+        return
+
+    if value['status'] in ['archived']:
```

Comment:
is not level defined below?

Reply:
you're right, fixed. surprised this didn't error out